File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] answer = new int[2];

        for (int i = 0; i < nums.length; i++) {
            for (int j = 0; j < nums.length; j++) {
                if (nums[i] + nums[j] == target && i != j) {
                    answer[0] = j;
                    answer[1] = i;
                }
            }
        }
        return answer;
    }
}

File: Data Structures & Algorithms/subarray-sums-divisible-by-k/submission-1.py
class Solution:
    def subarraysDivByK(self, nums: List[int], k: int) -> int:
        prefix = [0 for _ in range(len(nums) + 1)]
        total = 0
        for i in range(len(nums)):
            prefix[i] = total
            total += nums[i]
        prefix[-1] = total

        dic = defaultdict(int)
        count = 0
        for i in range(len(prefix)):
            count += dic[prefix[i] % k]
            dic[prefix[i] % k] += 1
        return count
        